File: Core/Entities/Quote.cs
using System;
using System.Collections.Generic;

namespace Core.Entities
{
  public class Quote
    {
        public decimal Discount { get; set; }
        public Service Service { get; set; }
        public decimal TotalPrice { get; set; }
        public decimal TotalPriceExVat { get; set; }
        public decimal TotalVat { get; set; }
        public decimal UnitPrice { get; set; }
        public decimal VatRate { get; set; }
        public DateTime Collection { get; set; }
        public DateTime CutOff { get; set; }
        public DateTime EstimatedDeliveryDate { get; set; }

        public List<AvailableExtra> AvailableExtras { get; set; }

        public Links Links { get; set; }

        public List<Quote> Quotes { get; set; }
    }
}

File: Client/Parcel2Go/src/app/quotes/models/quote.ts
import { AvailableExtras } from './availableExtras';
import { Service } from './service';

export class Quote {
  discount: string;
  service: Service;
  totalPrice: number;
  totalPriceExVat: number;
  totalVat: number;
  unitPrice: number;
  vatRate: number;
  collection: Date;
  cutOff: Date;
  estimatedDeliveryDate: Date;
  availableExtras: AvailableExtras[];
}
